compiler/testData/diagnostics/tests/controlFlowAnalysis/initializationInLocalViaExplicitThis_before.fir.kt

```diff
@@ -10,9 +10,9 @@ class Test {
 
         run {
             // Not sure do we need diagnostic also here
-            this@Test.str = "B"
+            <!VAL_REASSIGNMENT!>this@Test.str<!> = "B"
```

Comment:
`UseCallsInPlaceEffect` is enabled since 1.3, so having the same results with `..._after.fir.kt` is expected.